classifier.go

```diff
@@ -0,0 +1,97 @@
+package slinguist
+
+import (
+	"math"
+
+	"gopkg.in/src-d/simple-linguist.v1/internal/tokenizer"
+)
+
+// GetLanguageByClassifier takes in a content and a list of candidates, and apply the classifier's Classify method to
+// get the most probably language. If classifier is null then DefaultClassfier will be used.
+func GetLanguageByClassifier(content []byte, candidates []string, classifier Classifier) string {
+	if len(content) == 0 {
+		return OtherLanguage
+	}
+
+	if classifier == nil {
+		classifier = DefaultClassifier
+	}
+
+	languages := make([]string, 0, len(candidates))
+	for _, candidate := range candidates {
+		lang := GetLanguageByAlias(candidate)
+		if lang != OtherLanguage {
+			languages = append(languages, lang)
+		}
+	}
+
+	scores := classifier.Classify(content, languages)
+	language := getLangugeHigherScore(scores)
+	return language
+}
+
+func getLangugeHigherScore(scores map[string]float64) string {
+	var language string
+	higher := -math.MaxFloat64
+	for lang, score := range scores {
+		higher = math.Max(higher, score)
```

Comment:
I thought to accuraly compare float64 it was better option use this function. Should I change it anyway?